editor/settings/editor_settings.cpp

```diff
@@ -580,18 +580,20 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	// Theme
 	EDITOR_SETTING_BASIC(Variant::BOOL, PROPERTY_HINT_ENUM, "interface/theme/follow_system_theme", false, "")
-	EDITOR_SETTING_BASIC(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/preset", "Default", "Default,Breeze Dark,Godot 2,Gray,Light,Solarized (Dark),Solarized (Light),Black (OLED),Custom")
+	EDITOR_SETTING_BASIC(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/style", "Default", "Default,Classic")
+	EDITOR_SETTING_BASIC(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/preset", "Default", "Default,Classic,Breeze Dark,Godot 2,Gray,Light,Solarized (Dark),Solarized (Light),Black (OLED),Custom")
 	EDITOR_SETTING_BASIC(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/spacing_preset", "Default", "Compact,Default,Spacious,Custom")
 	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "interface/theme/icon_and_font_color", 0, "Auto,Dark,Light")
-	EDITOR_SETTING_BASIC(Variant::COLOR, PROPERTY_HINT_NONE, "interface/theme/base_color", Color(0.2, 0.23, 0.31), "")
-	EDITOR_SETTING_BASIC(Variant::COLOR, PROPERTY_HINT_NONE, "interface/theme/accent_color", Color(0.41, 0.61, 0.91), "")
+	EDITOR_SETTING_BASIC(Variant::COLOR, PROPERTY_HINT_NONE, "interface/theme/base_color", Color(0.14, 0.14, 0.14), "")
```

Comment:
I know this is meant to be almost a 1:1 port of Minimal Theme (this color corresponds to `#242424`, but I'd like to compare with other engines and popular color themes, so:

- Unreal Engine: `#2f2f2f`
- GameMaker: `#323232`
- Unity: `#383838`
- CryEngine: `#393939`
- Cocos2d-x: `#474747`

I made a previous comment about popular text editor color themes and would like to use them as an example too. But neutral colors are important, so I got their RGB means instead:

- Solarized Dark (base03): `#292929`
- Xcode Dark: `#2c2c2c`
- One Dark Pro/Dracula/GitHub/Tomorrow: `#2d2d2d`
- Solarized Dark (base02): `#2f2f2f`
- Nord: `#363636`

Personally, I would bump this up at least to `#292929`.

Reply:
> * Unreal Engine: `#2f2f2f`
> * GameMaker: `#323232`
> * Unity: `#383838`
> * CryEngine: `#393939`
> * Cocos2d-x: `#474747`

For me they look like black, black, black, black, and black🤣